src/platform/utilities/feature-toggles/helpers.js

```diff
@@ -0,0 +1,28 @@
+const Cookies = require('js-cookie');
+
+// Reads Google Analytics ID if available. Otherwise use a random string as a token.
+function createTokenFromCookie(cookieName) {
+  return Cookies.get(cookieName)
+    ? Cookies.get(cookieName)
+    : Math.random()
+        .toString(36)
+        .substring(2, 15) +
+        Math.random()
+          .toString(36)
+          .substring(2, 15);
+}
+
+// Gets and sets the cookie and then returns the flipper ID as a string
+function getFlipperId() {
+  const cookieName = 'FLIPPER_ID';
+
+  // Create a Flipper cookie if it doesn't exist
+  if (!Cookies.get(cookieName)) {
```

Comment:
Could simplify this a bit (remove my comments if you decide to use this):

```
// check my syntax- been a while since i tried this 
import uniqueId from 'lodash/uniqueId';

// set constants in the module context near import statements 
const cookieName = 'FLIPPER_ID';

// use default values to make this easier to unit test
function getFlipperId(_uniqueID = uniqueId, _cookieName = cookieName) {
  
const cookieValue = document.cookie[_cookieName] || _uniqueID();
// doesn't matter if we always set the cookie
document.cookie[_cookieName] = `${_cookieName}=${cookieValue}`;

return cookieValue;
```
no if statement required and it only needs two unit tests.

Reply:
I didn't port over the expiration date but I'm in favor of using it.